apps/sim-core/packages/core/.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
```

Comment:
Do we need all of these here or are they covered by parent directory ignore files? It seems like some of these, at least, should be more global.

Reply:
Seems like it, yeah. I'll see what makes sense to clean up.